### PR TITLE
studies/dedup: fix typos

### PR DESCRIPTION
**Pull request by cassella**
Including one in some filenames.

### CODE REVIEW

**Comment by cassella** [comment]
@mppf I assume these filenames were meant to have the 'u' in them?

start_test test/studies/dedup passes.

File: test/studies/dedup/dedup-distributed-ints.chpl [hunk 1]
```diff
@@ -61,7 +61,7 @@ proc main(args:[] string)
   sort(hashAndFileId);
 
   if verbose then
-    writeln("Outputing Duplicates");
+    writeln("outputting Duplicates");
```
Comment (ben-albrecht) [inline test/studies/dedup/dedup-distributed-ints.chpl]: Was lower-casing 'outputting' intentional here?

Comment (mppf) [inline test/studies/dedup/dedup-distributed-ints.chpl]: no

**Comment by mppf** [comment]
@cassella - if you want to correct the spelling errors in the test names, just git mv each .chpl and .good file and make sure the start_test still works.

File: test/studies/dedup/dedup-distributed-ints.chpl (hunk 1 above)
Comment (cassella) [inline test/studies/dedup/dedup-distributed-ints.chpl]: Whoops, nope, sorry.  I'll fix that up.

**Comment by cassella** [comment]
That's what I did.  :)

**Comment by mppf** [comment]
Right! for some reason I can't read the diffs correctly this morning. Anyway, thanks.

**Pull request merged**